src/resources/js/dependency.js

```diff
@@ -124,7 +124,8 @@
 
 					var active_class       = selectors.active.replace( '.', '' );
 					// Allows us to check a disabled dependency
-					var allowDisabled     = $dependent.data( 'dependencyCheckDisabled' ) || $dependent.is( '[data-dependency-check-disabled]' );
+					var allowDisabled      = $dependent.data( 'dependencyCheckDisabled' ) || $dependent.is( '[data-dependency-check-disabled]' );
+					var alwaysVisible      = $dependent.data( 'dependencyAlwaysVisibile' ) || $dependent.is( '[data-dependency-always-visible]' );
```

Comment:
use `snake_case` instead of `camelCase` (unless there's a hyper-specific reason not to?)

Reply:
On JavaScript we are going with `camelCase`.

Reply:
Oh damn—was not familiar with that standard change. Approving this PR, then!